ims/ostis_tech/lib_ostis/section_library_of_reusable_components_kb/section_subject_domain_of_users/relations/nrel_language_used_by_the_user.scs

```diff
@@ -0,0 +1,53 @@
+nrel_language_used_by_the_user
+<- sc_node_norole_relation;
+<- relation;
+<- binary_relation;
+<- oriented_relation;
+<- antireflexive_relation;
+<- asymmetric_relation;
+<- antitransitive_relation;
+=>nrel_main_idtf:
+	[используемый пользователем язык*]
+	(* <- lang_ru;; *);
+	[language used by the user*]
+	(* <- lang_en;; *);
+=> nrel_first_domain: concept_user;
+=> nrel_second_domain: languages;
+=> nrel_definitional_domain: 
+	... 
+	(*
+	<= nrel_combination: 
+		{
+		concept_user;
+		languages
+		};;
+	*);
+<- rrel_key_sc_element:
+	...
+	(*
+	=> nrel_main_idtf: 
+		[Опр. (используемый пользователем язык*)] 
+		(* <- lang_ru;; *);
+		[Def. (language used by the user*)]
+		(* <- lang_en;; *);;
+	<-  definition;;
+	<= nrel_sc_text_translation: 
+		...
+		(*
+		-> rrel_example: 
+			[Используемый пользователем язык* - бинарное отношение, связывающее пользователя и язык, который он использует для общения.]
```

Comment:
For communication with who?
In different systems user can use different languages, we should say about it, this language is only for this system